### ISSUE TITLE
Add misc regions that are in this ticket

### ISSUE THREAD

**Issue by jackiekazil** [unitedstates/agency-regions#3]
I have a misc list of regions lying around that need to be added to the repo. To add to the repo, do the following.... 
- Take screenshot of the map at the url
- Rename to use agency + sub-agency. Example "dot-faa.png" ... if no sub-agency, then just top level, ie "dot.png"
- Add them to the region-maps folder
- Add a line item to the csv that includes the source from where you got it. 

Region into to add:
- http://www.hhs.gov/iea/regional/
- http://www.consumerfinance.gov/blog/meet-our-regional-directors/

**Comment by neilmartis** [comment]
Hi Jackie I have found some maps for DOE,State and DOA where should I add them.I can't find the excel sheet to add them

**Comment by jackiekazil** [comment]
@neilmartis what format are they in? do you have images or actual shp files? 
If just images, then add image [here](https://github.com/unitedstates/agency-regions/tree/master/region-maps) and then document it [here](https://github.com/unitedstates/agency-regions/blob/master/region_maps.csv).

**Comment by neilmartis** [comment]
Hi Jackie I have an idea of what you were building since this data is going to be collected.If there is a map of usa we could plot the data on the maps for every agency using a drop down dynamically for the department and agencies and this data can also be used for other purposes.Am I right.

**Comment by jackiekazil** [comment]
Yes... ultimately it a perfect world we would have the data points for the
regions stored and not images.

On Mon, Mar 14, 2016 at 12:07 PM, Neil Martis notifications@github.com
wrote:

> Hi Jackie I have an idea of what you were building since this data is
> going to be collected.If there is a map of usa we could plot the data on
> the maps for every agency using a drop down dynamically for the department
> and agencies and this data can also be used for other purposes.Am I right.
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/unitedstates/agency-regions/issues/3#issuecomment-196386940
> .

## 

Jacqueline Kazil | @jackiekazil

**Comment by neilmartis** [comment]
Its a good project for me to work on I haven't done anything like this so your guidance will be helpful

**Comment by neilmartis** [comment]
I have done some work with rails & php not much with java and python

**Comment by neilmartis** [comment]
python with gis would be the right way to begin right

**Comment by neilmartis** [comment]
Sorry about that Jackie

**Comment by jackiekazil** [comment]
Yeah… that would be awesome!
Not sure if there is a way to automate, but there are two different ways to
extract the boundaries.

   -

   Acquire shp files from agencies
- Most accurate way
  
  ##   - But time consuming
  
  Extract the boundaries using qgis
- Not as accurate, because you can’t zoom to a very detailed level
  - Not as time consuming

In the 1970s, there was a GAO report that said if agencies had the shared
similar regional divides that could save a crapload of money. So, this was
why this repo got kicked off to start to compare programmatically how
different the regions were. (I did a search for the GAO report, but I
couldn’t find it, but I know it does exist.)
​

On Mon, Mar 14, 2016 at 1:12 PM, Neil Martis notifications@github.com
wrote:

> Sorry about that Jackie
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/unitedstates/agency-regions/issues/3#issuecomment-196416877
> .

## 

Jacqueline Kazil | @jackiekazil

**Comment by neilmartis** [comment]
I can look it up sorry busy with the job hunt.I did see ArcGIS with python the other day .Hi Jackie is this the GAO report you were speaking about http://www.gao.gov/products/GGD-85-17

**Comment by neilmartis** [comment]
sorry wrong report

**Comment by neilmartis** [comment]
Hi Jackie I started to collect the data and since it will be an interactive map I was thinking of using node.js leaflet.js and mongodb

**Comment by jackiekazil** [comment]
that work work.
The data should be in one repo, while the app in another. make sense?

On Fri, Mar 18, 2016 at 5:00 PM, Neil Martis notifications@github.com
wrote:

> Hi Jackie I started to collect the data and since it will be an
> interactive map I was thinking of using node.js leaflet.js and mongodb
> 
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> https://github.com/unitedstates/agency-regions/issues/3#issuecomment-198541017

## 

Jacqueline Kazil | @jackiekazil

**Comment by neilmartis** [comment]
yes Jackie

**Comment by neilmartis** [comment]
Hi Jackie I have added you the trello board.if you have a list of task defined me  on the board.I can work in a sprint weekly.

**Comment by jackiekazil** [comment]
Do you want to get together or do a virtual chat to discuss how this might
come together?

On Mon, Mar 21, 2016 at 3:01 PM, Neil Martis notifications@github.com
wrote:

> Hi Jackie I have added you the trello board.if you have a list of task
> defined me on the board.I can work in a sprint weekly.
> 
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> https://github.com/unitedstates/agency-regions/issues/3#issuecomment-199425742

## 

Jacqueline Kazil | @jackiekazil

**Comment by neilmartis** [comment]
Yes Jackie that would be good.As this week is Holy Week.Next week will be
good.

**Comment by neilmartis** [comment]
I am available via Skype or Google Hangout
